marketplace-service/src/test/java/com/axonivy/market/BaseSetup.java

```diff
@@ -46,6 +49,7 @@ public class BaseSetup {
   protected static final String MOCK_GROUP_ID = "com.axonivy.util";
   protected static final String MOCK_PRODUCT_NAME = "bpmn statistic";
   protected static final String MOCK_PRODUCT_JSON_FILE_PATH = "src/test/resources/product.json";
+  protected static final String MOCK_PRODUCT_JSON_FILE_PATH_NO_URL = "src/test/resources/product2.json";
```

Comment:
product2 is not good. Maybe you use another name. E.g. productWithoutURL or productMissingURL or ...